server/hooks/before_copy_to_cart_hook.js

```diff
@@ -1,5 +1,26 @@
 ReactionCore.MethodHooks.before('cart/copyCartToOrder', function (options) {
   const cart = ReactionCore.Collections.Cart.findOne(options.arguments[0]);
-  console.log('cart', cart)
+  if (cart) {
```

Comment:
Let's add some documentation style comments. This hook is semi-complicated and would be worth giving a little more documentation to.